python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -162,7 +179,7 @@ def setUp(self, gangadir=None, wipe_repo=None, extra_opts=[]):
         # This is called before each unittest
         if gangadir is None:
             import os
-            gangadir = os.path.join('$HOME/gangadir_testing', self.__class__.__name__)
+            gangadir = os.path.join('$HOME/gangadir_testing')  # TODO make separate dirs per test suite, `self.__class__.__name__`
```

Comment:
Ah, that sounds quite nice :)
In that case it's not worth investing too much time in making ```tearDownClass``` too complex